docs/guides/advanced_user/hydromt_python.rst

```diff
@@ -3,11 +3,12 @@
 Using HydroMT in Python
 =======================
 
-As HydroMT's architecture is very modular, it is possible to use HydroMT in Python
-without using the command line interface (CLI). With Python, you actually have access to
-a lot of internal functionalities. This can be useful if you want to for example:
+As HydroMT's architecture is modular, it is possible to use HydroMT as a python library
+without using the command line interface (CLI). Using the libary, you actually have
+access to many internal functionalities. This can be useful if you want to for example:
```

Comment:
Know this is from the previous version, but I'd rather not tell people "they have access to internal functionalities" we can say they have access to ones not necessarily available from the cli/yml but I think we should communicate something more along the lines of "internal means hands off or you're on your own"